src/postgres-data/buildData.ts

```diff
@@ -245,69 +229,23 @@ const getCollaboratorsData = async (
   return collabsData
 }
 
-const buildModelMapFromJson = (models: Model[]) => {
-  return new Map(
-    models.map((model) => {
-      return [model._id, model]
-    })
-  )
-}
-
 const getDrivedData = async (
   manuscriptID: string,
   projectID: string,
   data: Partial<state>
 ) => {
-  let storeData: Partial<state>
-
   if (!data.modelMap || !projectID) {
     return null
   }
 
-  const latestSnaphot = data?.snapshots?.length ? data.snapshots[0] : null
-  if (!latestSnaphot) {
-    const decoder = new Decoder(data.modelMap, true)
-    const doc = decoder.createArticleNode()
-    const ancestorDoc = decoder.createArticleNode()
-    storeData = {
-      snapshotID: null,
-      commitAtLoad: null,
-      ancestorDoc,
-      doc,
-    }
-  } else {
-    const modelsFromSnapshot = await getSnapshot(
-      projectID,
-      latestSnaphot.s3Id!
-    ).catch((e) => {
-      console.log(e)
-      throw new Error('Failed to load snapshot')
-    })
-    const snapshotModelMap = buildModelMapFromJson(
-      modelsFromSnapshot.filter(
-        (doc: any) => !doc.manuscriptID || doc.manuscriptID === manuscriptID
-      )
-    )
-    const unrejectedCorrections = (data.corrections as Correction[])
-      .filter(
-        (cor) =>
-          cor.snapshotID === data.snapshots![0]._id &&
-          cor.status.label !== 'rejected'
-      )
-      .map((cor) => cor.commitChangeID || '')
-
-    const commitAtLoad =
-      findCommitWithChanges(data.commits || [], unrejectedCorrections) || null
-    const decoder = new Decoder(snapshotModelMap, true)
-    const doc = decoder.createArticleNode() as ManuscriptNode
-    const ancestorDoc = decoder.createArticleNode() as ManuscriptNode
-    storeData = {
-      snapshotID: data.snapshots![0]._id,
-      modelMap: snapshotModelMap,
-      commitAtLoad,
-      ancestorDoc,
-      doc,
-    }
+  const decoder = new Decoder(data.modelMap, true)
+  const doc = decoder.createArticleNode()
+  const ancestorDoc = decoder.createArticleNode()
+  const storeData: Partial<state> = {
+    snapshotID: null,
```

Comment:
That may need a further cleanup later on, to make sure no components are using it and then removing it from the store type.